CHANGELOG.md

```diff
@@ -0,0 +1,41 @@
+# libhoney Changelog
+
+## 1.7.1 2019-03-29
+
+Documentation updates only.
+
+## 1.7.0 2019-02-10
+
+Improvements
+
+- JSON encoder now defaults to string encoding of types previously not handled by the default encoder (Such as datetime).
+
+Security Updates
+
+- Updates example Django app to 1.8.19 in response to CVE-2017-7233
+
+## 1.6.2 2018-10-09
+
+Improvements
+
+- Adds default HTTP timeout of 10s. Previously, connections to the Honeycomb API would never timeout.
```

Comment:
Why was this a problem that connections would never time out? (Playing devil's advocate for a bit more detail on user impact.)